<ui version="4.0" >
 <class>EldoraPPI</class>
 <widget class="QDialog" name="EldoraPPI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="PPI" native="1" name="ppiFor" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>3</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>knob</string>
           </property>
           <property name="whatsThis" >
            <string>A knob that you can turn.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ColorBar" name="colorBarFor" >
           <property name="minimumSize" >
            <size>
             <width>60</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="forwardGroupBox" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>3</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="PPI" native="1" name="ppiAft" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>3</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>knob</string>
           </property>
           <property name="whatsThis" >
            <string>A knob that you can turn.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ColorBar" name="colorBarAft" >
           <property name="minimumSize" >
            <size>
             <width>60</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="aftGroupBox" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>3</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pauseRunButton" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Pause</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ringColorButton" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Ring/grid color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backgroundButton" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Background color</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveImage" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Save Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ringsCheck" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Rings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="gridCheck" >
            <property name="minimumSize" >
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string>Grid</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="zoomIn" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Zoom in</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="zoomOut" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Zoom out</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel2_4" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Zoom:</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="LCDNumberZoomFactor" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>1</width>
                <height>0</height>
               </size>
              </property>
              <property name="smallDecimalPoint" >
               <bool>false</bool>
              </property>
              <property name="segmentStyle" >
               <enum>QLCDNumber::Flat</enum>
              </property>
              <property name="value" stdset="0" >
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="1" >
             <widget class="QPushButton" name="panDown" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text" >
               <string>v</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QPushButton" name="viewReset" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text" >
               <string>o</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="panLeft" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text" >
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" >
             <widget class="QPushButton" name="panRight" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text" >
               <string>></string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QPushButton" name="panUp" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text" >
               <string>^</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Elevation</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="elevation" >
          <property name="text" >
           <string>0.0</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorBar</class>
   <extends>QWidget</extends>
   <header>ColorBar.h</header>
  </customwidget>
  <customwidget>
   <class>PPI</class>
   <extends>QWidget</extends>
   <header>PPI.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
